### PR TITLE
recover some explanations on the readme

### PR DESCRIPTION
**Pull request by staltz**
Follow up for #61

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,11 +34,13 @@ classDef root fill:#8338ec,stroke:#8338ec,color:#fff;
...
-whose sole responsibility is to announce (point to) sub-feeds that you publish content to.
+How scuttlebutt works with metafeeds - each device now has a `root` metafeed,
+whose sole responsibility is to announce (point to) subfeeds that you publish
+content to. A subfeed can also be a metafeed, which then allows the existence of
+"sub-subfeeds".
 
-This means that when you first meet a peer you can replicate their `root` feed
-and (having discovered their subfeeds), replicate just their `aboutMe` and `contacts` feeds
+This means that when you first meet a peer you can replicate their `root` metafeed
+and, having discovered their subfeeds, replicate just their `aboutMe` and `contacts` feeds
```
Comment (mixmix) [inline README.md:43]: Maybe we should emphasize that these subfeeds are just examples of what you might want.

 replicate just the subfeeds you care about e.g. Aboutme...

File: README.md [hunk 2]
```diff
@@ -69,35 +71,59 @@ Add this plugin like this:
...
+  // Find an existing feed
   const isFeed = (feed) => feed.feedpurpose === 'aboutMe'
-  // details for creating a sub-feed feed if it doesn't already exist
+  // Details for creating a subfeed if it doesn't already exist:
   const details = { feedpurpose: 'aboutMe', feedformat: 'classic' }
 
-  sbot.metafeeds.findOrCreate(rootFeed, isFeed, details, (err, aboutMeFeed) => {
-    console.log(aboutMeFeed)
-
+  sbot.metafeeds.findOrCreate(root, isFeed, details, (err, aboutMeFeed) => {
```
Comment (mixmix) [inline README.md:96]: I liked rootFeed, makes it clear that it's the same as aboutMeFeed

Not gonna die, just think it makes it clearer than root

File: README.md [hunk 3]
```diff
@@ -69,35 +71,59 @@ Add this plugin like this:
...
     // ...
   })
 })
 ```
 
-Once you have these the *FeedDetail* object `aboutMeFeed` you can publish to that feed:
+The `details` argumentis a *FeedDetails* object and it always requires
+`feedpurpose` (any string) and `feedformat` (either `'classic'` for normal
+ed25519 SSB feeds, or `'bendybutt-v1'` if you want a metafeed). The `isFeed`
+function also takes a *FeedDetails* object as input.
```
Comment (mixmix) [inline README.md:105]: Nice!

File: README.md [hunk 4]
```diff
@@ -69,35 +71,59 @@ Add this plugin like this:
...
-    console.log(aboutMeFeed)
-
+  sbot.metafeeds.findOrCreate(root, isFeed, details, (err, aboutMeFeed) => {
     // ...
   })
 })
 ```
 
-Once you have these the *FeedDetail* object `aboutMeFeed` you can publish to that feed:
+The `details` argumentis a *FeedDetails* object and it always requires
```
Comment (mixmix) [inline README.md:102]: Typo : argumentis

**Review by mixmix — APPROVED** [review]
Only one typo. Rest is opinions. Looks better i agree.

One thing I find a bit wearisome and confusing is what is a metafeed. What's the difference between a feed and a meta feed? 
I think we should say (if we don't already), that a metafeed does pointing to other feeds (and nothing else), while all other application logic is kept in "normal" feeds. 

Then say the root feed is a meta-feed (points to other subfeeds). Then after that just call it the root feed. Or root feed (topmost meta feed).

I think the confusing aspect is that meta means "higher" abstraction to me while root means... kinda lower. It feels like mixed metaphors to me when I first read it. I'm now getting used to it, but we want it very clear for beginners

**Comment by staltz** [comment]
Thanks for the review! I made the typo correction plus some metafeed conceptual explanation.

**Pull request merged**